undebt/cmd/main.py

```diff
@@ -57,21 +57,21 @@ def _write_result_text(result_text, path, dry_run):
 def _handle_arguments():
     parser = argparse.ArgumentParser(prog='undebt')
     parser.add_argument(
-        '--input', '-i', metavar='path', action='append',
-        help='paths to files or directories (searched recursively for extension) to be modified (if not passed uses stdin)')
+        'paths', nargs='*', metavar='PATH',
+        help='paths to files or directories (searches for extension recursively) to be modified; '
+        'uses stdin if not passed')
     parser.add_argument(
-        '--pattern', '-p', metavar='path', action='append', required=True,
+        '--pattern', '-p', metavar='PATH', action='append', required=True,
         help='paths to pattern definition files')
```

Comment:
nit: shouldn't this read "path to pattern definition file" rather than "paths ... to files" ?